conda_recipe_tools/cli/find_latest.py

```diff
@@ -45,19 +45,42 @@ def main():
     if not args.no_header:
         print('package_name,latest_version')
     for name in names_to_check:
+        lookup_fail = True
         info = pkg_info.get(name, {})
         update_type = info.get('update_type', 'pypi')
         extra = info.get('update_extra', {})
         try:
             latest_version = find_latest_version(name, update_type, extra)
+            lookup_fail = False
         except FeatureNotFound as e:
+            # this will raise is bs4 cannot use lxml as a feature to parse html
             raise e
         except ConnectionError as e:
+            # this will happen when there is a timeout
             latest_version = 'could_not_reach_url'
-        except:
+        except NoURLError as e:
+            # this will happen when there is no url specified
+            latest_version = 'url_is_none'
+        except NoVersionError as e:
+            # this will happen when we cannot find a suitable url
+            latest_version = 'no_versions_found'
+        except NoCustomLookupError as e:
+            # this happens when lookup_type is custom but we don't have it
```

Comment:
For clarity, can we change this comment to:
_This happens when lookup_type is 'custom' in ad_pkg_info but we don't have it implemented yet in conda_recipe_tools/find_version.py_